library/certs.c

```diff
@@ -40,24 +40,24 @@
 /* BEGIN FILE string macro TEST_CA_CRT_EC_PEM tests/data_files/test-ca2.crt */
```

Comment:
Please forward-port the extra changes to the two certs. There is a good chance that, before the end of 2.28 LTS, we will create a separate test repository which will be used to test all versions. At that point, we won't be able to have different files in `tests/data_files`.